game-schema-d4.json

```diff
@@ -88,31 +88,33 @@
                 "type": "string",
                 "enum":
                 [
-                    "RPG",
-                    "Open Source",
-                    "Adventure",
                     "Action",
-                    "Puzzle",
+                    "Adventure",
+                    "Arcade",
+                    "Card Game",
+                    "Educational",
+                    "Fighting",
+                    "gbprinter",
+                    "Interactive Fiction",
+                    "Open Source",
                     "Platform",
+                    "Puzzle",
+                    "Racing",
+                    "Rhythm",
+                    "RPG",
+                    "Shooter",
+                    "Simulation",
+                    "Sports",
+                    "Strategy",
+                    "Survival",
+                    "Visual Novel",
                     "gbajam21",
                     "gbcompo21",
                     "gbcompo21-shortlist",
-                    "gbprinter",
```

Comment:
This file got auto-formatted probably and introduced a bunch of unrelated changes, can you please disable the formatter and only commit the additional tag ?

Reply:
Done!